src/background.ts

```diff
@@ -539,6 +539,17 @@ async function start() {
   }
   store.set("engineSettings", engineSettings);
 
+  const recentlyUsedProjects = store.get("recentlyUsedProjects");
```

Comment:
ここ起動シーケンスでメンテが大変なので、コメントで区切っておけると･･･！
```suggestion
  // 「最近開いたプロジェクト」のうちファイルが存在しないものをフィルター
  const recentlyUsedProjects = store.get("recentlyUsedProjects");
```